content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -70,272 +76,443 @@ $ export REMOTE_CLUSTER_NAME=<your remote cluster name>
+    NAME                                               TYPE                                  DATA   AGE
+    default-token-m9nnj                                kubernetes.io/service-account-token   3      2m37s
+    istio-ca-secret                                    istio.io/ca-root                      5      18s
+    istio-reader-service-account-token-prnvv           kubernetes.io/service-account-token   3      2m31s
+    istiod-service-account-token-z2cvz                 kubernetes.io/service-account-token   3      2m30s
+    {{< /text >}}
+
+### Mesh admin steps
+
+Now that Istio is up and running, a mesh administrator only needs to deploy and configure services in the mesh,
+including gateways, if needed.
+
+#### Deploy a sample application
+
+1. Create, and label for injection, the `sample` namespace on the remote cluster:
+
+    {{< text bash >}}
+    $ kubectl create --context="${CTX_REMOTE_CLUSTER}" namespace sample
+    $ kubectl label --context="${CTX_REMOTE_CLUSTER}" namespace sample istio-injection=enabled
+    {{< /text >}}
+
+1. Deploy the `helloworld` (`v1`) and `sleep` samples:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/helloworld/helloworld.yaml -l service=helloworld -n sample --context="${CTX_REMOTE_CLUSTER}"
+    $ kubectl apply -f samples/helloworld/helloworld.yaml -l version=v1 -n sample --context="${CTX_REMOTE_CLUSTER}"
+    $ kubectl apply -f samples/sleep/sleep.yaml -n sample --context="${CTX_REMOTE_CLUSTER}"
+    {{< /text >}}
+
+1. Wait a few seconds for the `helloworld` and `sleep` pods to be running with sidecars injected:
+
+    {{< text bash >}}
+    $ kubectl get pod -n sample --context="${CTX_REMOTE_CLUSTER}"
+    NAME                             READY   STATUS    RESTARTS   AGE
+    helloworld-v1-5b75657f75-ncpc5   2/2     Running   0          10s
+    sleep-64d7d56698-wqjnm           2/2     Running   0          9s
+    {{< /text >}}
+
+1. Send a request from the `sleep` pod to the `helloworld` service:
+
+    {{< text bash >}}
+    $ kubectl exec --context="${CTX_REMOTE_CLUSTER}" -n sample -c sleep \
+        "$(kubectl get pod --context="${CTX_REMOTE_CLUSTER}" -n sample -l app=sleep -o jsonpath='{.items[0].metadata.name}')" \
+        -- curl -sS helloworld.sample:5000/hello
+    Hello version: v1, instance: helloworld-v1-5b75657f75-ncpc5
+    {{< /text >}}
+
+#### Enable gateways
+
+1. Enable an ingress gateway on the remote cluster:
+
+    {{< text bash >}}
+    $ cat <<EOF > istio-ingressgateway.yaml
+    apiVersion: operator.istio.io/v1alpha1
+    kind: IstioOperator
+    spec:
+      profile: empty
+      components:
+        ingressGateways:
+        - namespace: external-istiod
+          name: istio-ingressgateway
+          enabled: true
+      values:
+        gateways:
+          istio-ingressgateway:
+            injectionTemplate: gateway
+    EOF
+    $ istioctl install -f istio-ingressgateway.yaml --context="${CTX_REMOTE_CLUSTER}"
```

Comment:
This command will currently fail. Latest master build fails to inject the gateway template because the deployment has `istio.io/rev: default` (instead of no `istio.io/rev` label), which the sidecar-injector webhook doesn't currently allow.

@Monkeyanator I understand (from @howardjohn) that you have an upcoming fix for this. Is that correct?

Reply:
Yes, that's correct. I was going to hold off until the default revision work merged, but since that might be delayed a bit, I'll get an implementation up ASAP

Reply:
Great, thanks!

Reply:
it should work with https://github.com/istio/istio/pull/31344 anyways?

Reply:
Oh, yes @howardjohn, looks like it is already working with #31344. Thanks.